.circleci/config.yml

```diff
@@ -95,21 +100,32 @@ jobs:
             ansible-galaxy install -r ansible/requirements.yml
       - run:
           name: Test ansible setup playbook
+          environment:
+            - ANSIBLE_CALLBACK_WHITELIST: profile_tasks
```

Comment:
I see [key](https://docs.ansible.com/ansible/2.6/reference_appendices/config.html#envvar-ANSIBLE_CALLBACK_WHITELIST) and [value](https://docs.ansible.com/ansible/2.6/plugins/callback/profile_tasks.html) documented. Ok.